samples/SocketsSample/project.json

```diff
@@ -55,7 +55,7 @@
   },
 
   "scripts": {
-    "precompile": [ "tsc --project ../../src/Microsoft.AspNetCore.SignalR.Client.TS/ --out wwwroot/js/signalr-client.js" ],
+    "precompile": [ "../../bin/nodejs/node_modules/.bin/tsc --project ../../src/Microsoft.AspNetCore.SignalR.Client.TS/ --out wwwroot/js/signalr-client.js" ],
```

Comment:
Alternatively we should just install TypeScript globally during build... which seems... ok, I guess :)